planet/Decoration.ocd/Village.ocd/BridgeRustic.ocd/Script.c

```diff
@@ -19,3 +19,5 @@ protected func Construction()
 
 	return true;
 }
+
+local Visibility = VIS_Editor;
```

Comment:
That would make it visible in the editor, but invisible in the game. Sounds unintended.

Reply:
The object is transparent anyway.
I figured this would perhaps allow for optimization.

Reply:
Transparent of invisible?

Reply:
Yes?

Reply:
If it is invisible, why have it visible in the editor? If it is just has transparent but is otherwise visible, then it should not have VIS_Editor

Reply:
I've removed it.